### DISCUSSION TITLE
How should we cite tarski?

### DISCUSSION THREAD

**Discussion by haz** [aig-upf/tarski#119] [Q&A]
I'd like to cite it...and, thus, make you all rich & famous! ;)

**Comment by miquelramirez** [comment]
Hello @haz,

this is a revision the way we cited it in our recent "Approximate Novelty Search" paper:

```
@misc{tarski:github:18,
  author = {Guillem Franc\'{e}s and Miquel Ramirez and Collaborators},
  title = {Tarski: An {AI} Planning Modeling Framework},
  year = {2018},
  publisher = {{GitHub}},
  journal = {{GitHub} repository},
  howpublished = {\url{https://github.com/aig-upf/tarski}}
}
```

> make you all rich & famous! ;)

![image](https://user-images.githubusercontent.com/444457/126576509-22813089-6175-4a54-ac06-3af2feaf2873.png)

**Reply by haz** [reply]
You get but one wish!

Gracias.
